### PR TITLE
Tasks views

### LINKED ISSUE #20: Task's views.
New, Edit, Index, Show views, containing my tasks header, task name, new task button, other tasks header, task name, join in button

### PR DESCRIPTION
**Pull request by tprochniewicz**
closes #20

### CODE REVIEW

File: app/views/tasks/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+      <td><%= @task.description %></td>
+    </tr>
+    <tr>
+      <td><h4>Deadline</h4></td>
+      <td><%= @task.due_date %></td>
+    </tr>
+    <tr>
+      <td><h4>Status</h4></td>
+      <td>
+        <% if @task.done == true %>
```
Comment (pospieszynski) [inline app/views/tasks/show.html.erb]: if @task.done
 is enough

File: spec/views/tasks/_form.html.erb_spec.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
+require "rails_helper"
+
+RSpec.describe "tasks/_form", type: :view do
+  let(:user) { FactoryGirl.create(:user) }
+
+  before do
+    sign_in(user)
+
```
Comment (pospieszynski) [inline spec/views/tasks/_form.html.erb_spec.rb]: empty line

File: app/views/tasks/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  </thead>
+
+  <tbody>
+    <% @tasks.each do |task| %>
+      <tr>
+        <td><%= task.title %></td>
+        <td><%= task.due_date %></td>
+        <td><%= task.description[0, 100] %>...</td>
+        <td>
+          <% if task.done == true %>
```
Comment (pospieszynski) [inline app/views/tasks/index.html.erb]: task.done is enough

File: app/views/tasks/show.html.erb (hunk 1 above)
Comment (pospieszynski) [inline app/views/tasks/show.html.erb]: same here

File: app/views/tasks/show.html.erb [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+              <li> <%= user.name %> <%= user.surname %></li>
+            <% end %>
+          </ul>
+        <% end %>
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+<% if @task.done == false %>
```
Comment (pospieszynski) [inline app/views/tasks/show.html.erb]: maybe change to unless?

**Comment by pospieszynski** [comment]
:+1: 
@gotar

File: app/views/tasks/_form.html.erb [hunk 5]
```diff
@@ -0,0 +1,27 @@
+<%= form_for([project, task], url: url) do |f| %>
+  <% if task.errors.any? %>
+    <div id="error_explanation">
+      <ul>
+        <% task.errors.full_messages.each do |message| %>
+          <li><%= message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
```
Comment (gotar) [inline app/views/tasks/_form.html.erb]: you can have global helper to shown form error, more reusable if neeeded

**Comment by gotar** [comment]
squash and it can go live

**Pull request merged**